### PR TITLE
 DWR-512: IRQ signal and led polarity added to devicetree

### PR DESCRIPTION
**Pull request by fid0did0**
Update to properly set the sms led active hi and to map the active
low IRQ generated by the si3210.

Signed-off-by: Giuseppe Lippolis <giu.lippolis@gmail.com>

### CODE REVIEW

File: target/linux/ramips/image/rt305x.mk [hunk 1]
```diff
@@ -296,6 +296,7 @@ TARGET_DEVICES += dir-620-d1
 
 define Device/dwr-512-b
   DTS := DWR-512-B
+  IMAGE_SIZE := 7700k
   DEVICE_TITLE := D-Link DWR-512 B
```
Comment (blogic) [inline target/linux/ramips/image/rt305x.mk]: this is an unrelated change. either update the description of the patch or remove this chunk

**Comment by pepe2k** [comment]
Hi @fid0did0,

IMHO, change in `target/linux/ramips/image/rt305x.mk ` should go to a separate commit with a proper description.

Cheers,
Piotr

**Comment by blogic** [comment]
patch subject is wrong

**Comment by fid0did0** [comment]
I split it into two different commits with differents commit message.
Let me know if I need to split the PR.

Bye.

**Comment by fid0did0** [comment]
dear @blogic I need to ask you about the correctness of my IRQ description.
Actually, the si3210 generate an IRQ on the gpio1.
In the DT shall I report the gpio number (1) or the irq number attached to the gpio controller (6)?

Bye.

**Comment by pepe2k** [comment]
Hi @fid0did0,

Both of your commits are missing `ramips:` prefix.

Cheers,
Piotr

**Comment by fid0did0** [comment]
Hi @pepe2k,
I added the prefix accordingly in the commits subject.

Bye.
